openfl/component/collaborator/collaborator.py

```diff
@@ -6,12 +6,21 @@
 from enum import Enum
 from logging import getLogger
 from time import sleep
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import NamedTuple
 from typing import Tuple
+from typing import Union
 
+from numpy import ndarray
+
+from openfl.component.assigner.tasks import Task
```

Comment:
We use here `Task`, but there are other places where we can replace `Any` on `Task` to refine the annotation. For example, inside `aggregator.py` file.